easybuild/easyconfigs/p/Platypus/Platypus-0.8.1-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'Platypus'
+version = '0.8.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.well.ox.ac.uk/platypus'
+description = """Platypus is a tool designed for efficient and accurate variant-detection
+ in high-throughput sequencing data."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+# download requires registration, see http://www.well.ox.ac.uk/software-download-registration
+sources = ['%(name)s_%(version)s.tgz']
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('HTSlib', '1.3'),
+]
+
+unpack_options = '--strip-components=1'
+buildininstalldir = True
+
+options = {'modulename': 'platypusutils'}
+
+sanity_check_paths = {
+    'files': ['Platypus.py'],
```

Comment:
maybe I should create a symlink in `bin` to the main script though (since there's a lot of other cruft in the root dir with execution permissions enabled)